config/upgrade.go

```diff
@@ -34,7 +34,7 @@ func upgradeConfig(helper up.Helper) {
 	helper.Copy(up.Bool, "meowlnir", "dry_run")
 	helper.Copy(up.Str|up.Null, "meowlnir", "report_room")
 	helper.Copy(up.List, "meowlnir", "hacky_rule_filter")
-	helper.Copy(up.Bool, "meowlnir", "filter_local_invites")
```

Comment:
This should be moved to the correct place (under `antispam`) rather than removed